src/main/webapp/components/Export.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import * as FileSaver from 'file-saver';
+import * as XLSX from 'xlsx';
+import { formatExcelData } from "../util/TableUtil";
+
+export const Export = ({csvData, columns, fileName}) => {
+
+  const fileType = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet;charset=UTF-8';
+  const fileExtension = '.xlsx';
+
+  const exportToExcel = (csvData, columns, fileName) => {
+    let formatedCsvData = formatExcelData(csvData, columns);
+    let ws = XLSX.utils.json_to_sheet(formatedCsvData,{skipHeader:true});
+    const wb = { Sheets: { 'data': ws }, SheetNames: ['data'] };
+    const excelBuffer = XLSX.write(wb, { bookType: 'xlsx', type: 'array' });
```

Comment:
Can `bookType` be parameterized to match the file type?

Reply:
Done!